src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,20 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import { Todo } from '../../types';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    // eslint-disable-next-line
+    <article data-id={todo.id} className={classNames('TodoInfo', {'TodoInfo--completed': todo.completed})}>
```

Comment:
No need for eslint exceptions here
```suggestion
    <article
      data-id={todo.id}
      className={
        classNames('TodoInfo', { 'TodoInfo--completed': todo.completed })
      }
    >
```